### PR TITLE
Journalisierung PDF-Download

### LINKED ISSUE #2019: Journalisierung PDF-Download
Der PDF-Download muss auch journalisiert werden. Die PDF-Download URL zeigt im Moment direkt auf Bumblebee, dies muss so angepasst werden, dass man erst nach der journalisierung redirected wird.

### PR DESCRIPTION
**Pull request by deiferni**
Add a redirector view that creates journal entries when downloading PDFs from bumblebee.

Closes #2019.

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross plz have a look

File: opengever/journal/locales/de/LC_MESSAGES/opengever.journal.po [hunk 1]
```diff
@@ -35,172 +35,177 @@ msgid "label_comments"
...
 #. Default: "Participant removed: ${contact}"
-#: ./opengever/journal/handlers.py:673
+#: ./opengever/journal/handlers.py:683
 msgid "label_participant_removed"
 msgstr "Beteiligung von ${contact} entfernt"
 
+#. Default: "PDF downloaded"
+#: ./opengever/journal/handlers.py:498
+msgid "label_pdf_downloaded"
+msgstr "Download PDF"
```
Comment (phgross) [inline opengever/journal/locales/de/LC_MESSAGES/opengever.journal.po]: `Download ` is no translated --> `PDF heruntergeladen`

File: opengever/journal/tests/test_bumblebee_events.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def tearDown(self):
+        super(TestBumblebeeJournal, self).tearDown()
+        deactivate_bumblebee_feature()
+
+    def get_journal_entries(self, obj):
+        annotations = IAnnotations(obj)
+        data = annotations.get(JOURNAL_ENTRIES_ANNOTATIONS_KEY, [])
+        return data
+
+    def assert_journal_entry(self, action, title, entry):
```
Comment (phgross) [inline opengever/journal/tests/test_bumblebee_events.py]: The `assert_journal_entry` is already provided by the FunctionalTestCase, not necessary to copy.

File: opengever/bumblebee/events.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from five import grok
+from opengever.bumblebee.interfaces import IPDFDownloadedEvent
+from zope.component.interfaces import ObjectEvent
+
+
+class PDFDownloadedEvent(ObjectEvent):
+    """ The ObjectCheckedOutEvent is triggered when a object
```
Comment (lukasgraf) [inline opengever/bumblebee/events.py]: Copy & pasted docstring

Comment (lukasgraf) [inline opengever/bumblebee/events.py]: Copy & pasted docstring

**Comment by lukasgraf** [comment]
Otherwise looks good 👍

**Comment by phgross** [comment]
@lukasgraf updated ...

**Pull request merged**